cli/mender-artifact/main.go

```diff
@@ -94,6 +94,22 @@ func run() error {
 			Usage: "Full path to the state script(s). You can specify multiple " +
 				"scripts providing this parameter multiple times.",
 		},
+		/////////////////////////
+		// Version 3 specifics.//
+		/////////////////////////
+		cli.StringSliceFlag{
+			Name:  "artifact-name-depends",
+			Usage: "Sets the name(s) of the artifact(s) which this update depends upon",
+		},
+		// TODO - this flag, or device-type?
+		cli.StringSliceFlag{
+			Name:  "artifact-device-depends",
+			Usage: "Sets the name(s) of the device(s) which this update depends upon",
+		},
+		cli.StringFlag{
+			Name:  "group",
+			Usage: "Sets the artifact group",
+		},
```

Comment:
Two things:

* I would rename it to `provides-group`. It's still a nice sounding flag, and more similar to the spec, so less chance of confusion, if only among us.
* There should be a `depends-groups` as well, which should be a list aka `-t` which you can specify multiple times. This is not described super well in the spec (it's missing from the example), but you can see it in the list of members for artifact_depends.